src/WebJobs.Script/OutOfProc/Http/HttpFunctionInvocationDispatcher.cs

```diff
@@ -162,6 +161,7 @@ protected virtual void Dispose(bool disposing)
                 _logger.LogDebug("Disposing {HttpFunctionInvokeDispatcher}", nameof(HttpFunctionInvocationDispatcher));
```

Comment:
For something like this, where the value is static, I wouldn't use structured logging. It'll give someone a key/value pair of `HttpFunctionInvokerDispatcher=HttpFunctionInvokerDispatcher`... which isn't very useful. Put that `nameof()` inline instead. There may be other instances of this as well.